pymtl3/dsl/NamedObject.py

```diff
@@ -152,8 +152,13 @@ def __setattr_for_elaborate__( s, name, obj ):
       if isinstance( obj, NamedObject ):
         ud = obj._dsl
 
+        try:
+          ud.level      = sd.level + 1
+        except AttributeError:
+          raise PyMTLDeprecationError("\n- Some component in the hierarchy implements __init__ "
```

Comment:
To be clear, is it true that any legal PyMTL component that could ever be is NOT allowed to override `__init__`? Is there any possible use of that?

Reply:
hmmm maybe I can remove this